tracing-flame/src/lib.rs

```diff
@@ -235,6 +255,36 @@ where
             out: self.out.clone(),
         }
     }
+
+    /// Configures whether or not periods of time where no spans are entered
+    /// should be included
+    ///
+    /// Defaults to true
+    ///
+    /// Setting this feature to false can help with situations where you have
+    /// large periods of time spent idling or otherwise doing uninteresting work
+    /// where no spans are active. These empty samples can pollute the final
+    /// flamegraph and squish the work you care about into a much smaller
+    /// percentage of the overall graph, making it harder to navigate and in some
+    /// cases even hiding spans.
+    pub fn with_empty_samples(mut self, enabled: bool) -> Self {
+        self.config.empty_samples = enabled;
+        self
+    }
+
+    /// Configures whether or not spans from different threads should be
+    /// collapsed into one pool of events
+    ///
+    /// Defaults to false
```

Comment:
```suggestion
    /// Configures whether or not spans from different threads should be
    /// collapsed into one pool of events.
    ///
    /// Defaults to `false`.
```

Reply:
Also, not a huge fan of "pool of events", it's not super clear to me what exactly that means. Can we maybe reword this?